templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -6,5 +6,6 @@ <h1>Delete manufacturer?</h1>
     {% csrf_token %}
 
     <input type="submit" value="Yes" class="btn btn-danger">
+    <a href="{% url 'manufacturer_list' %}" class="btn btn-default">No</a>
   </form>
 {% endblock %}
```

Comment:
missed blank line

Reply:
I am sorry) I now you already tired to tell this(((